src/test_workflow/perf_test/perf_test_suite.py

```diff
@@ -20,16 +18,24 @@ def __init__(self, bundle_manifest, endpoint, security, current_workspace):
             f" -a {self.manifest.build.architecture} -p {self.current_workspace}"
         )
 
+        if test_results_path is not None:
+            self.command = (
+                f"pipenv run python test_config.py -i {self.endpoint} -b {self.manifest.build.id}"
+                f" -a {self.manifest.build.architecture} -p {test_results_path}"
+            )
+
+        print(self.command)
+
     def execute(self):
         try:
-            with WorkingDirectory(self.work_dir):
-                dir = os.getcwd()
-                subprocess.check_call("python3 -m pipenv install", cwd=dir, shell=True)
-                subprocess.check_call("pipenv install", cwd=dir, shell=True)
+            os.chdir(os.path.join(self.current_workspace, self.work_dir))
```

Comment:
Is there any reason why can't we use `WorkingDirectory` here?

Reply:
Updated to use `WorkingDirectory`